examples/configs/llm/hf_endpoint/README.md

```
@@ -1,7 +1,26 @@
 # HuggingFace Endpoint
 
+> **Framework requirement.** This example uses `engine: huggingface_endpoint`, which is served by the LangChain framework only.
+> Set `NEMOGUARDRAILS_LLM_FRAMEWORK=langchain` and install `pip install langchain langchain-community` before running.
+> The `parameters.model_kwargs` block in `config.yml` is also LangChain-specific and is rejected by the default framework.
+
 This configuration uses the HuggingFace [Inference Endpoints](https://huggingface.co/docs/inference-endpoints/index).
 
 First, create an endpoint following the instructions [here](https://huggingface.co/docs/inference-endpoints/guides/create_endpoint). Then, update the `endpoint_url` key in the `config.yml` file.
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)